views/js/pciManager/pciManager.js

```diff
@@ -82,180 +84,229 @@ define([
+                            off: {
+                                active: $li.hasClass('pci-disabled')
                             }
-                        }).on('on', function(){
-                            $li.removeClass('pci-disabled');
-                            $.getJSON(urls.enableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
-                                    listing[typeIdentifier].enabled = true;
-                                    self.trigger('pciEnabled', typeIdentifier);
-                                }
+                        })
+                            .on('on', function () {
+                                $li.removeClass('pci-disabled');
+                                $.getJSON(urls.enableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                    if (data.success) {
+                                        listing[typeIdentifier].enabled = true;
+                                        self.trigger('pciEnabled', typeIdentifier);
+                                    }
+                                });
+                            })
+                            .on('off', function () {
+                                $li.addClass('pci-disabled');
+                                $.getJSON(urls.disableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                    if (data.success) {
+                                        listing[typeIdentifier].enabled = false;
+                                        self.trigger('pciDisabled', typeIdentifier);
+                                    }
+                                });
                             });
-                        }).on('off', function(){
-                            $li.addClass('pci-disabled');
-                            $.getJSON(urls.disableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
-                                    listing[typeIdentifier].enabled = false;
-                                    self.trigger('pciDisabled', typeIdentifier);
-                                }
+
+                        buttonFactory({
+                            id: 'unregister',
+                            type: 'info',
+                            icon: 'bin',
+                            label: 'Delete',
+                            class: 'unregister',
+                            renderTo: pcibuttons
+                        })
+                            .on('click', function confirmDialog() {
+                                dialog({
+                                    class: 'icon-warning',
+                                    heading: __('Warning'),
+                                    message: __('You are about to delete the Portable Custom Interaction %s from the system.', typeIdentifier),
```

Comment:
I thought of one more thing, could you try to wrap `<strong>%s</strong>` tags to make the name of the PCI bold, like in the mockup?